clang/test/SemaCXX/requires-expression-with-capture-variable.cpp

```diff
@@ -0,0 +1,25 @@
+// RUN: %clang -fsyntax-only -std=c++20 -Xclang -verify %s
+
+// expected-no-diagnostics
+
+auto GH69307_Func_1() {
+  constexpr auto b = 1;
+  return [&](auto c) -> int
```

Comment:
Can you do a similar test with NO capture here?  To show that they aren't needed.